src/sonic-frr-mgmt-framework/frrcfgd/frrcfgd.py

```diff
@@ -1791,9 +1802,9 @@ class BGPConfigDaemon:
                    (['local_asn', '+local_as_no_prepend',
                      '+local_as_replace_as'],               '{no:no-prefix}neighbor {} local-as {} {:no-prepend} {:replace-as}'),
                    (['admin_status', '+shutdown_message'],  '{no:no-prefix}neighbor {} shutdown {:shutdown-msg}', ['false', 'true']),
-                   ('local_addr',                           '{no:no-prefix}neighbor {} update-source {}'),
+                   ('update_source',                           '{no:no-prefix}neighbor {} update-source {}'),
                    ('name',                                 '{no:no-prefix}neighbor {} description {}'),
-                   (['ebgp_multihop', '+ebgp_multihop_ttl'],'{no:no-prefix}neighbor {} ebgp-multihop {}', ['true', 'false']),
+                   (['ebgp_multihop', '+ebgp_multihop_ttl'],'{no:no-prefix}neighbor {} ebgp-multihop {}'),
```

Comment:
Could you please explain this change?

Reply:
The current usage is 

https://github.com/eddieruan-alibaba/sonic-mgmt/blob/eruan-frrcfgd-cfg/tests/srv6/7nodes_cisco/PE1.json#L95

I don't know how to add "true" and "false" and don't understand the need to have this "true" and "false"

Reply:
Please revert this change and use as following for ebgp

```
 "default|2064:300::1f": {
            "admin_status": "true",
            "update_source" :"2064:100::1d",
            **"ebgp_multihop": "true",
            "ebgp_multihop_ttl":  "255",**
            "capability_ext_nexthop": "true",
            "local_addr" :"2064:100::1d",
            "local_asn" : "64600",
            "asn": "64602",
            "name": "PE3"
        },

```

Reply:
Revert back previous change and updated json file.